packages/components/src/Icon/Icon.component.js

```diff
@@ -27,7 +27,7 @@ export const SVG_TRANSFORMS = {
 	'flip-vertical': theme['flip-vertical'],
 };
 
-export const TRANSFORMS = Object.keys(SVG_TRANSFORMS);
+const TRANSFORMS = Object.keys(SVG_TRANSFORMS);
```

Comment:
This is not used outside, because it's not usable. It's an array of strings 'rotate-90', ... . This is to group them for proptypes.
In practice you won't import that and use TRANSFORM[3] to pass the value in your jsx.
This is purely internal for proptypes.